Dockerfile

```diff
@@ -1,10 +1,6 @@
 FROM node:6-onbuild
 MAINTAINER Trent Oswald (@therebelrobot) <trentoswald@therebelrobot.com>
-RUN npm run install:clean --production
```

Comment:
I can do that work here, if you want, but it would require tying the `npm start` script to the `npm run install:clean --production` script